Client.xcodeproj/project.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -92,10 +83,10 @@
     {
       "identity" : "rust-components-swift",
       "kind" : "remoteSourceControl",
-      "location" : "https://github.com/mozilla/rust-components-swift.git",
+      "location" : "https://github.com/ecosia/rust-components-swift.git",
       "state" : {
-        "revision" : "90a4bb34e6205d25fa6f67a04f71d9a6f31a91f4",
-        "version" : "96.1.3"
+        "branch" : "91.6.3_Glean_dependency_removed",
```

Comment:
Since you created an ecosia fork of the repository, wouldn't it make sense to update main of it instead of using a separate branch?

Reply:
Awesome catch 👌 . Was expecting a comment along these lines as it does make sense. I thought the same way. However, I wasn't sure about a few things I wanted to clarify before merging this code in our new `main` branch on that forked library.

- the end goal was achieved 👍 (no CI errors)
- what's the concrete process when creating our `main` branch from a forked repo (as their `main` that we pulled is based on newer version and we don't want other changes pulled along)
- is there any concrete value in doing it as other dependencies (`Dip` for instance) are based on specific commits

I'm totally up to make it our `main` on the forked library or even pushing a patch of this current version.
Mentioning @ecotopian so that we can close the loop on it 👌

Reply:
Ah, my bad, I hadn't realised you based on an earlier version than main has since we are not using the latest one, you are totally right, that does change things 😅

I would say ideally we would have forked the needed version and only update it when updating FF as well... But I guess that is no longer possible since it is already updated, so I believe your solution of using a branch to be appropriate 🤔

Reply:
@d4r1091 ah I see you're right. Their ´main´  branch is already quite advanced, I see. Technically we could force push to main having 96.1.3 as head. Honestly I don't have a too strong opinion on this. It's also the first time we do it, that we fork a dep of FF. So up to us to find a process that fits.

Reply:
when working with a branch as SPM reference, it happens that the commit changes when the branch gets new commits. Thus it adds noise to the Client app, where this will show up in the git change set of the Package.resolved file. Thus a tag is more concrete.

Reply:
Totally up for tagging 👍 .